as-controller-board-status/main.go

```diff
@@ -31,20 +32,41 @@ func main() {
 		os.Exit(1)
 	}
 
-	boardStatus := functions.CheckBoardStatus{
-		DoorClosed:    true, // Set default door state to closed
-		Configuration: new(functions.ControllerBoardStatusAppSettings),
-		Service:       service,
+	subscriptionClient := service.SubscriptionClient()
+	if subscriptionClient == nil {
+		lc.Errorf("error notification service missing from client's configuration")
+		os.Exit(1)
 	}
 
+	config := new(functions.ControllerBoardStatusAppSettings)
 	// Retrieve & parse the required application settings into a proper
 	// configuration struct
-	err := utilities.MarshalSettings(appSettings, boardStatus.Configuration, false)
+	err := utilities.MarshalSettings(appSettings, config, false)
```

Comment:
We can replace use of `utilities.MarshalSettings` with use of custom configuration. Please create an issue.

Reply:
issue #44 created